docs/sources/datasources/loki.md

```diff
@@ -54,92 +48,65 @@ You can use a debug section to see what your fields extract and how the URL is i
+| `Query expression` | Loki query expression, check out the [LogQL documentation](https://grafana.com/docs/loki/latest/logql/).                                                                                                    |
+| `Query type`       | Choose the type of query you would like to run. An instant query queries against a single point in time. We are using "To" time from the time range. A range query queries over the selected range of time. |
+| `Line limit`       | Upper limit for number of log lines returned by query. The default is the Maximum lines limit set in Loki settings.                                                                                         |
+| `Legend`           | Available only in Dashboard. Controls the name of the time series, using name or pattern. For example `{{hostname}}` is replaced with the label value for the label `hostname`.                             |
 
-The following label matching operators are currently supported:
+### Log browser
 
-- `=` exactly equal.
-- `!=` not equal.
-- `=~` regex-match.
-- `!~` do not regex-match.
+// TODO David
 
-Examples:
+## Querying with Loki
 
-- `{name=~"mysql.+"}`
-- `{name!~"mysql.+"}`
+There are two types of LogQL queries:
 
-The [same rules that apply for Prometheus Label Selectors](https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors) apply for Loki Log Stream Selectors.
+- **Log queries** that return the contents of log lines.
+- **Metric queries** that extend log queries and calculate sample values based on the content of logs from a log query.
 
-Another way to add a label selector is in the table section. Click **Filter** beside a label to add the label to the query expression. This even works for multiple queries and will add the label selector to each query.
-
-### Search Expression
-
-After writing the Log Stream Selector, you can filter the results further by writing a search expression. The search expression can be just text or a regex expression.
-
-Example queries:
-
-- `{job="mysql"} |= "error"`
-- `{name="kafka"} |~ "tsdb-ops.*io:2003"`
-- `{instance=~"kafka-[23]",name="kafka"} != "kafka.server:type=ReplicaManager"`
-
-Filter operators can be chained and will sequentially filter down the expression. The resulting log lines will satisfy every filter.
+### Log Queries
 
-**Example**
+Querying and displaying log data from Loki is available via [Explore]({{< relref "../explore" >}}), and with the [logs panel]({{< relref "../panels/visualizations/logs-panel.md" >}}) in dashboards. Select the Loki data source, and then enter a [LogQL](https://grafana.com/docs/loki/latest/logql/#log-queries) query to display your logs.
 
-`{job="mysql"} |= "error" != "timeout"`
+A log query consists of two parts: **log stream selector**, and a **log pipeline**. For performance reasons you need to start by choosing a log stream by selecting a log label.
```

Comment:
Why are the terms bold? If they are not on the UI, they should not be bold. Apply italic if you want to emphasize them.

Reply:
Removed bold. I guess, they don't need to be bold.